src/ssvc/_mixins.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+"""
+file: _basics
+author: adh
+created_at: 9/20/23 4:51 PM
+"""
+from dataclasses import dataclass
+
+from dataclasses_json import dataclass_json
+
+
+@dataclass_json
+@dataclass(kw_only=True)
+class _Versioned:
+    """
+    Mixin class for versioned SSVC objects.
+    """
+
+    version: str = "0.0.0"
+
+
+@dataclass_json
+@dataclass(kw_only=True)
+class _Namespaced:
+    """
+    Mixin class for namespaced SSVC objects.
+    """
+
+    namespace: str = "ssvc"
+
+
+@dataclass_json
+@dataclass(kw_only=True)
+class _Keyed:
```

Comment:
This mixin class was added so that we can remove the key from the base class that DecisionPointGroup inherits from.